src/components/social/facebook/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import FacebookLogin from 'react-facebook-login';
+import LoginFacebook from '../../../actions/socialLogin/FacebookActions';
+import store from '../../../store';
+
+class FacebookOption extends Component {
+  responseFacebook = (response) => {
+    const FacebookToken = response.accessToken;
+    store.dispatch(LoginFacebook(FacebookToken));
+    document.location.href = '/successpage';
+  };
+
+  render() {
+    return (
+      <div>
+        <FacebookLogin
+          appId="2097710087203447"
```

Comment:
Please @Rhytah, could you consider setting this in the environmental variables for this appIdd.